variables.tf

```diff
@@ -409,6 +409,13 @@ variable "public_ssh_key" {
   default     = ""
 }
 
+variable "rbac_aad" {
+  type        = bool
+  description = "(Optional) Is Azure Active Directory ingration enabled?"
+  default     = false
```

Comment:
I dont like the idea of enabling this by default, so I have set this to `false`.
But this is a breaking change, because this will disable the `azure_active_directory_role_based_access_control` block.

I am in doubt if `true` is a better default to avoid breaking changes.

Cc:
@lonegunmanb

Reply:
As agreed in our meeting, we will set the default value to true because it is the most secure value for the users.